srcpkgs/msbuild/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'msbuild'
+pkgname=msbuild
+version=15.6
+revision=1
+_xamarin_rel="xamarinxplat.2018.01.17.14.14-0xamarin2+ubuntu1404b1"
+build_style="fetch"
+short_desc="Xamarin/Mono fork of the Microsoft Build system"
+maintainer="Aloz1 <kno0001@gmail.com>"
+license="MIT"
+homepage="https://github.com/mono/msbuild"
+distfiles="http://download.mono-project.com/repo/ubuntu/pool/main/m/${pkgname}/${pkgname}_${version}+${_xamarin_rel}_all.deb"
+checksum=d5254973c3bf3d297bd1fb256d44f68bb4b28617c26f14018935e5962e40fc49
+
+do_extract() {
+	echo $PWD
```

Comment:
what purpose do these two lines serve?

Reply:
They don't have any purpose, must have missed them (was 2AM when I made the PR :/) They've now been removed.